democracylab/emails.py

```diff
@@ -222,20 +223,19 @@ def send_volunteer_application_email(volunteer_relation, is_reminder=False):
     email_template = HtmlEmailTemplate()\
         .subheader("Opportunity Information:")\
         .text_line("Title: {role}".format(role=role_details.display_name))\
-        .text_line("Organization: {project}".format(project=project.project_name))\
-        .text_line("Date: {currentdate}".format(currentdate=models.DateTimeField(auto_now_add=True, null=True)))\
+        .text_line("Organization: {projectname}".format(projectname=project.project_name))\
+        .text_line("Date: {currentdate}".format(currentdate=datetime_to_string(timezone.now(), DateTimeFormats.DATE_LOCALIZED)))\
         .subheader("Volunteer Information:")\
         .text_line("Name: {firstname} {lastname}".format(
             firstname=user.first_name,
             lastname=user.last_name))\
-        .text_line("Email: {email}".format(email=Html.a(href='mailto:' + user.email, text=user.email)))\
+        .text_line("Email: {email}".format(email=Html.a(href='mailto:{useremail}'.format(useremail=user.email), text=user.email)))\
         .text_line("Zip: {zip}".format(zip=user.postal_code))\
```

Comment:
There's no conditional formatting here (and I'm not really sure why we ask for this field / put it in the email, since it's not like I've memorized every Seattle zip code to know if a volunteer is "local" or not?) -- is the `is_reminder` if statement a good pattern to follow for conditionally showing or hiding this? I think having a blank Zip: field is pretty awkward.

Reply:
We'll want to format things a bit differently here.  Essentially, we don't want to include the line at all if there's no zip, so we can do something like

email_template = HtmlEmailTemplate()\
// all the lines up till the zip line
if user.postal_code:
  email_template = email_template.text_line("Zip: {zip}".format(zip=user.postal_code))
// everything else